js/components/configuration.html

```diff
@@ -29,27 +39,27 @@
 						<th>Record Counts (RC / DRC)</th>
 						<th>Incidence</th>
 					</thead>
-					<tbody data-bind="foreach:$data.sources">
+					<tbody data-bind="foreach:$component.sources()">
 						<tr>
 							<td><i class="fa" data-bind="if:!initialized, attr: { title : error }, css: { 'fa-check-circle': initialized, 'fa-exclamation-circle': !initialized }"></i>
 								<span data-bind="text:error"></span>
 							</td>
-							<td data-bind="text:sourceName + ' [' + sourceKey + ']'"></td>
+							<td><a href="#" data-bind="text:sourceName + ' [' + sourceKey + ']', click: $component.selectSource"></a></td>
```

Comment:
The source names on the configuration screen should only have a link when the user has rights to view or edit the source details. Furthermore, we should consider using the source_id and not the source_key here for consistency with the way we're doing things in the rest of the app.